[2023-10-22T16:43:12.163+0800] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_notification.send_message_to_slack manual__2023-10-22T08:38:08.565874+00:00 [queued]>
[2023-10-22T16:43:12.168+0800] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_notification.send_message_to_slack manual__2023-10-22T08:38:08.565874+00:00 [queued]>
[2023-10-22T16:43:12.169+0800] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-10-22T16:43:12.176+0800] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): send_message_to_slack> on 2023-10-22 08:38:08.565874+00:00
[2023-10-22T16:43:12.184+0800] {standard_task_runner.py:57} INFO - Started process 3628 to run task
[2023-10-22T16:43:12.188+0800] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'stock_notification', 'send_message_to_slack', 'manual__2023-10-22T08:38:08.565874+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/stock_notification.py', '--cfg-path', '/tmp/tmp6hwr3jsc']
[2023-10-22T16:43:12.191+0800] {standard_task_runner.py:85} INFO - Job 232: Subtask send_message_to_slack
[2023-10-22T16:43:12.214+0800] {task_command.py:416} INFO - Running <TaskInstance: stock_notification.send_message_to_slack manual__2023-10-22T08:38:08.565874+00:00 [running]> on host 6a6999315df3
[2023-10-22T16:43:12.249+0800] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='joshua881117@gmail.com' AIRFLOW_CTX_DAG_OWNER='Joshua Lin' AIRFLOW_CTX_DAG_ID='stock_notification' AIRFLOW_CTX_TASK_ID='send_message_to_slack' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T08:38:08.565874+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T08:38:08.565874+00:00'
[2023-10-22T16:43:12.261+0800] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/slack_sdk/web/internal_utils.py:416: UserWarning: client.files_upload() may cause some issues like timeouts for relatively large files. Our latest recommendation is to use client.files_upload_v2(), which is mostly compatible and much stabler, instead.
  warnings.warn(message)

[2023-10-22T16:43:13.006+0800] {python.py:194} INFO - Done. Returned value was: None
[2023-10-22T16:43:13.023+0800] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=stock_notification, task_id=send_message_to_slack, execution_date=20231022T083808, start_date=20231022T084312, end_date=20231022T084313
[2023-10-22T16:43:13.077+0800] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T16:43:13.097+0800] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
